src/generators/schema-generator.php

```diff
@@ -128,6 +128,10 @@ public function generate( Meta_Tags_Context $context ) {
 				$graph_pieces = [ $graph_pieces ];
 			}
 
+			if ( ! is_array( $graph_pieces ) ) {
```

Comment:
I fixed it because I ran into it when developing. Therefore I didn't checked why it happens. I don't think a changelog entry isn't that valuable because it hasn't been released yet.